### DISCUSSION TITLE
关于 My staking on Camellia

### DISCUSSION THREAD

**Discussion by pzjboy0924** [tearust/Chinese-wiki#5] [General]
插槽和滑点这两个的意思
CML上面slot是有位置的，所有者占据第0个位置，然后投资的人往依次下排，越靠前的位置，分的收益就越多
所以你在投资的时候，如果有人抢，你可能就占不到你想占的位置。比如你想占1，另外一人比你抢先一步把1占了，你滑点如果选0的话，这个操作就失败了，如果你这里选1，就是说你可以容忍被往后挤一位。

**Comment by kevingzhang** [comment]
我觉得你可以直接发到wiki里面

**Reply by pzjboy0924** [reply]
我不知道具体放到哪个目录里

**Comment by kevingzhang** [comment]
感谢贡献

**Comment by kevingzhang** [comment]
没事儿你就这样写. 随后我们整理的时候给出link即可

**Comment by kevingzhang** [comment]
我给你加在wiki这里了 https://github.com/tearust/Chinese-wiki/wiki/%E4%BB%8E%E6%8A%95%E8%B5%84%E8%80%85%E8%A7%92%E5%BA%A6%E4%B8%8A%E7%9C%8B%E5%BF%85%E9%A1%BB%E4%BA%86%E8%A7%A3%E7%9A%84%E5%9F%BA%E7%A1%80%E7%9F%A5%E8%AF%86
你可以过去看
